packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/datebox.mask.tests.js

```diff
@@ -12,6 +12,9 @@ const { test, module } = QUnit;
 
 const CLEAR_BUTTON_AREA_CLASS = 'dx-clear-button-area';
 
+const DROP_EVENT_NAME = 'drop';
+
+
```

Comment:
```suggestion

```

Reply:
Fixed in [f0b0bf5f2a89e12ddc6f0d76ad534fe221a08aef].